phpunit/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.service.opg.digital/opguk/php-fpm
+
+WORKDIR /tmp
+
+ADD  docker/phpunit-4.8.35.phar phpunit-4.8.35.phar
+
+RUN chmod +x phpunit-4.8.35.phar
+RUN mv phpunit-4.8.35.phar /usr/local/bin/phpunit
+
+RUN apt-get install php5-mcrypt
```

Comment:
The newer opguk/php-fpm containers should include php5-mcrypt. Can you check please?